### ISSUE TITLE
Multiple Preferred audio language

### ISSUE THREAD

**Issue by subexx** [jellyfin/jellyfin-android#1434]
Is it possible to add an option to select multiple audio languages under Preferred audio language like Emby does?

**Comment by BotBlake** [comment]
What exactly would you expect this feature to do?
preferred language currently SHOULD automatically select the first audio-track with that language.

What would happen with two preferred languages?
Should the seccond one just be used as a failback?

**Comment by subexx** [comment]
You're correct. However, when my primary language isn’t available in movies with multiple audio tracks, it defaults to the first track, which often isn’t my preferred regional language.

Adding support for a second preferred audio language as a fallback, would ensure better language selection for content with non-English media.

**Comment by BotBlake** [comment]
Since you want to ad a full setting for language preferences you should note that this can not be done purely on Client side.
The server is responsible for choosing which audio/subtitle track the client plays by default.
The Clients then simply build UI to change these preferences. 

If you do want this feature on the Jellyfin server, please create a feature request on
https://features.jellyfin.org in case there is not already one.

I'm sorry for the confusion and the long waiting time.
I hope I was abled to help you with your issue!

Kind regards!
